browser/profiles/brave_renderer_updater.cc

```diff
@@ -80,14 +77,12 @@ BraveRendererUpdater::BraveRendererUpdater(
                           base::Unretained(this)));
 #endif
 
-#if BUILDFLAG(ENABLE_WIDEVINE)
```

Comment:
that's been added to remove a crash on linux arm64 https://github.com/brave/brave-core/commit/409b7a0ca96275b16a38560e7272f9d98a877caa as we didn't support widevine on that platform. From my understanding we support it currently right? In that case we can remove it in that place as well https://github.com/brave/brave-core/commit/409b7a0ca96275b16a38560e7272f9d98a877caa#diff-ee85b0dca2867e754c68f6e4b21fa42452a7cf72ff50a9a7e15380e93d43608aR198